### PR TITLE
AsyncFunction: restore fd_pipes initialization

### PR DESCRIPTION
**Pull request by aggstam**
Due to the missing pw, UserQuery prompt fails when using --ask flag on emaint

### CODE REVIEW

**Comment by aggstam** [comment]
For reference, this is the breaking commit: 1c0b5ff6aabe99224b07f7fc2e09e4cb6c9c70c4

**Comment by aggstam** [comment]
(Fixing tests)

File: lib/portage/util/_async/AsyncFunction.py [hunk 1]
```diff
@@ -25,6 +25,8 @@ class AsyncFunction(ForkProcess):
 
     def _start(self):
         pr, pw = multiprocessing.Pipe(duplex=False)
+        self.fd_pipes = {} if self.fd_pipes is None else self.fd_pipes
+        self.fd_pipes[pw] = pw
```
Comment (zmedico) [inline lib/portage/util/_async/AsyncFunction.py:29]: We actually need to pass stdin via fd_pipes. See https://github.com/gentoo/portage/pull/1145.

Comment (aggstam) [reply lib/portage/util/_async/AsyncFunction.py:29]: aha good to know, I will close this PR since its obselete, good job!
KR

**Pull request closed without merging**